common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -440,7 +453,7 @@ class @Problem
         @el.addClass 'showed'
         @el.find('.show').attr('disabled', 'disabled')
         @updateProgress response
-        window.SR.readElts(answer_text)
+        window.SR.readText(gettext('The answer to the problem has been shown. Navigate through the problem to read the answer text.'))
```

Comment:
"Text" is not necessary. :-) 
@cahrens Can we please go with "Answers to this problem are now shown. Navigate through the problem to review it with answers inline."